components/Scoped/Announcement/FAnnouncementCard.jsx

```diff
@@ -52,27 +54,29 @@ export const FAnnouncementCard = ({
               title={data.description}
               style={{ marginVertical: sizes.MARGIN_5 }}
               color={colors.DARK_GRAY}
-              ellipsizeMode="tail"
-              numberOfLines={2}
               size={fonts.HEADING_NORMAL}
               weight={fonts.HEADING_WEIGHT_REGULAR}
+              ellipsizeMode="tail"
+              numberOfLines={2}
             />
             <FHeadingWithIcon
               icon={icons.LOCATION_OUTLINE}
               iconColor={colors.PRIMARY}
               title={data.locationName}
               titleStyle={styles.text}
               titleColor={colors.DARK_GRAY}
-              numberOfLines={2}
               iconPlacement={placements.LEFT}
               iconSize={sizes.ICON_20}
               titleSize={fonts.HEADING_SMALL}
               titleWeight={fonts.HEADING_WEIGHT_REGULAR}
+              containerStyle={{ width: '90%' }}
```

Comment:
słabo to działa na mniejsze urządzenia, potrzebujesz to jakoś inaczej uzależnic, nie od width.  Zamiast tego spróbuj poszukać przyczyny dlaczego samo nie umie zobaczyc że powinno sie przyciać
![image](https://user-images.githubusercontent.com/61546640/164891578-473771d7-f608-4394-aa64-0392363f3ae8.png)
Musisz bardziej zwracać uwage na RWD :D